diff --git a/news/scripts/run_tasks.py b/news/scripts/run_tasks.py
index e259ce2..d169a9b 100644
--- a/news/scripts/run_tasks.py
+++ b/news/scripts/run_tasks.py
@@ -73,15 +73,15 @@ class Worker:
         shifts = [TimeShift(timezone) for timezone in timezones]
         while True:
             logging.info('Sleep 300s ...')
-            sleep(300)
+            sleep(60)
             current_shifts = [shift.current for shift in shifts]
             if all(shift == TimeShift.REST for shift in current_shifts):
-                _rested_time += 300
-                if _rested_time >= 10800:
+                _rested_time += 60
+                if _rested_time >= 600:
                     break
             elif all(shift >= TimeShift.NIGHT for shift in current_shifts):
-                _rested_time += 300
-                if _rested_time >= 3600:
+                _rested_time += 60
+                if _rested_time >= 600:
                     break
             else:
                 # if any(shift >= TimeShift.WORKING for shift in shifts):
